core-feature-pack/src/main/resources/modules/system/layers/base/org/eclipse/jgit/main/module.xml

```diff
@@ -29,15 +29,18 @@
 
     <resources>
         <artifact name="${org.eclipse.jgit:org.eclipse.jgit}"/>
+        <artifact name="${org.eclipse.jgit:org.eclipse.jgit.ssh.apache}"/>
     </resources>
 
     <dependencies>
         <module name="org.slf4j" />
-        <module name="org.slf4j.impl" />
+        <module name="org.slf4j.impl"/>
         <module name="javax.api"/>
         <module name="com.jcraft.jsch"/>
         <module name="com.jcraft.jzlib"/>
+        <module name="org.apache.sshd"/>
         <module name="com.googlecode.javaewah"/>
         <module name="org.apache.httpcomponents.core"/>
+        <module name="org.bouncycastle" services="import" export="true"/>
```

Comment:
Why export=true? The org.jboss.as.controller module is the only one that depends on org.eclipse.jgit and it shouldn't need to access any bouncycastle module classes or resources.

Reply:
I've removed the `export="true"`, thanks!